src/components/20-molecules/file-upload/index.js

```diff
@@ -158,6 +160,13 @@ class AXAFileUpload extends LitElement {
     }
   }
 
+  fireCustomChangeEvent() {
+    this.onChange({
+      detail: this.files,
+    });
+    fireCustomEvent('change', this.files, this);
```

Comment:
Why doing it like this? Better register a 'change' listener on the axa-input-file tag and just pass on that event. Or is that not possible?

Reply:
The change listener would only be called if the file was uploaded via the input file. And this happens max. one time (the first time a file gets uploaded)

Reply:
All other usecases like drag & drop or file deletion would be ignored.

Reply:
Best would be to artificially reroute the changes to the input-file button. The input file button would always be there (visible or not) and contain the current state of the file upload. This keeps it simple in terms of single source of truth.

Reply:
But the file upload as a component is now the singel source of truth. In my opinion it's not nice that a input file for example triggers an event onDeletion, because nowbody expects it to do so.

Reply:
I agree with Noah, this is fine and readable. But since there has been some discussion, a comment motivating this way of doing things by mentioning drag/drop and deletion would help!

Reply:
Ok, I understand that my idea would not be a proper solution. Sad that `input type=file` has such a bad api.